packages/client/TODO.md

```diff
@@ -53,10 +53,10 @@ Client manages the token lifecycle and storage
 
 ### Query profile
 
-- [Q] client.profile.fetch(request) (by id or handle, forSources) or client.profiles.fetchOne()
-- [PQ] client.profile.fetchAll(request) (by ids, ownedBy, handles, whoMirroredPublicationId)
-- [Q] client.profile.fetchRecommended()
-- [PQ] client.profile.mutualFollowers({ viewingProfileId, yourProfileId })
+- [x] client.profile.fetch(request) (by id or handle, forSources) or client.profiles.fetchOne()
+- [x] client.profile.fetchAll(request) (by ids, ownedBy, handles, whoMirroredPublicationId)
+- [x] client.profile.fetchRecommended()
```

Comment:
the method you added is called `allRecommended` (not `fetchRecommended`) and there are more changes like that, which is ok, but could you also update to the new naming in the TODO file?